integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA3336HostHeaderValuePortCheckTestCase.java

```diff
@@ -93,9 +93,13 @@ public void stop() throws Exception {
     private void applyProperty(File srcFile, String key, String value) throws Exception {
         File destinationFile = new File(srcFile.getName());
         Properties properties = new Properties();
-        properties.load(new FileInputStream(srcFile));
+        FileInputStream fis = new FileInputStream(srcFile);
+        properties.load(fis);
         properties.setProperty(key, value);
-        properties.store(new FileOutputStream(destinationFile), null);
+        fis.close();
+        FileOutputStream fos = new FileOutputStream(destinationFile);
+        properties.store(fos, null);
         serverConfigurationManager.applyConfigurationWithoutRestart(destinationFile);
+        fos.close();
```

Comment:
It's better if we could handle this inside a try-catch-finally block.

Reply:
Fixed. Please refer latest commit